session/session.go

```diff
@@ -1256,13 +1268,37 @@ func (s *session) SetProcessInfo(sql string, t time.Time, command byte, maxExecu
 	s.processInfo.Store(&pi)
 }
 
-func (s *session) ExecuteInternal(ctx context.Context, sql string) (recordSets []sqlexec.RecordSet, err error) {
+func (s *session) ExecuteInternal(ctx context.Context, sql string, args ...interface{}) (recordSets []sqlexec.RecordSet, err error) {
```

Comment:
1: could it be compatible with executing prepared statement via ExecuteInternal?
2: could we take `SELECT * from t  WHERE id LIKE '%name'` into consideration?

Reply:
1. We only escape/format arguments, disallow multi statements, using utf8mb4. And everything else works like before.
2. You need to write like `%%name` or `%?"), "%name"`. Otherwise, we need a partial SQL parser to know the specifier is actually in a string. I think it is better not to be too "smart" here.

Reply:
ok, since we only use it in our internal SQL, I think we can control it by %%name manually.